source/docs/software/kinematics-and-odometry/differential-drive-odometry.rst

```diff
@@ -0,0 +1,80 @@
+Differential Drive Odometry
+===========================
+A user can use the differential drive kinematics classes in order to perform odometry. WPILib contains a ``DifferentialDriveOdometry`` class that can be used to track the position of a differential drive robot on the field.
```

Comment:
What is odometry? This needs to be explained. Assume the user knows 0 about autonomous.

Reply:
A more detailed explanation of what odometry is was explained in the introductory article. I can add a link to that article here though.